easybuild/easyconfigs/u/util-linux/util-linux-2.41-GCCcore-14.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'util-linux'
+version = '2.41'
+
+homepage = 'https://www.kernel.org/pub/linux/utils/util-linux'
+
+description = "Set of Linux utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['%s/v%%(version_major_minor)s' % homepage]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['c014b5861695b603d0be2ad1e6f10d5838b9d7859e1dd72d01504556817d8a87']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('pkgconf', '2.3.0'),
+]
```

Comment:
Needs to change to using Meson instead of configure/make

The configure/make setup doesn't detect that it sometimes needs to link ipcmk with pthread which the meson does.

Reply:
well i did my best to convert it to meson, the options are not the same. Also, we are getting shared libraries now, which is fine i guess.